// Task 1: Finally or just the code?

/* try {
  work work
} catch (err) {
  handle errors
} finally {
  cleanup the working space
} */

/* try {
  work work
} catch (err) {
  handle errors
}

cleanup the working space */

// Same unless try..catch is used inside function and that function have a return statement. In this case 2 variant won't work
